dotCMS/src/integration-test/java/com/dotcms/security/apps/AppsAPIImplTest.java

```diff
@@ -33,6 +33,7 @@
 import com.dotmarketing.exception.DotSecurityException;
 import com.dotmarketing.exception.InvalidLicenseException;
```

Comment:
I'm not sure if we have all the tests for method validateForSave covered here

Reply:
and what makes you think so?

Reply:
there are 3 test for import-export
- Test_Create_Offending_Secrets_Then_Export_Them_Then_Import_Then_Save
- Test_Create_Secrets_Then_Export_Them_Then_Import_Then_Save
- Test_Import_With_Non_Valid_License

Reply:
validateForSave is covered here Test_Create_Secrets_Then_Export_Them_Then_Import_Then_Save